pkg/network/multus/nad.go

```diff
@@ -68,16 +68,12 @@ func GetNetworkToResourceMap(virtClient kubecli.KubevirtClient, vmi *v1.VirtualM
 			return map[string]string{}, fmt.Errorf("failed to locate network attachment definition %s", nadNamespacedName.String())
 		}
 
-		networkToResourceMap[network.Name] = GetResourceNameForNetwork(netAttachDef)
+		networkToResourceMap[network.Name] = ResourceNameFromNetAttachDef(netAttachDef)
 	}
 
 	return networkToResourceMap, nil
 }
 
-func GetResourceNameForNetwork(network *networkv1.NetworkAttachmentDefinition) string {
-	resourceName, ok := network.Annotations[ResourceNameAnnotation]
-	if ok {
-		return resourceName
-	}
-	return "" // meaning the network is not served by resources
+func ResourceNameFromNetAttachDef(netAttachDef *networkv1.NetworkAttachmentDefinition) string {
+	return netAttachDef.Annotations[ResourceNameAnnotation]
```

Comment:
This looks redundant now, it has no noticeable abstraction.

Reply:
Removed.